extensions/amp-story/1.0/amp-story.css

```diff
@@ -880,31 +880,32 @@ amp-story .amp-video-eq,
   /* 224px = the width of a narrow phone (320px) - tap-safe margins (48px * 2) */
   min-width: 224px !important;
   max-width: 400px !important;
-  /* Properties overrideable by user */
-  --interactive-accent-color: #005AF0;
-  --interactive-background-color: white;
-  /* Shared properties by components */
-  --interactive-prompt-text-color: white !important;
+	/* Properties overrideable by user */
+	--interactive-accent-color: #005AF0;
+	--interactive-prompt-background-color: var(--interactive-accent-color);
+	--interactive-prompt-text-color: white;
+	--interactive-prompt-alignment: initial;
+	/* Shared properties by components */
+	--interactive-option-accent-color: var(--interactive-accent-color);
+	--interactive-option-background-color: white !important;
   --interactive-option-text-color: #5F6368 !important;
   --interactive-theme-border: #DADCE0 !important;
   --interactive-theme-shading: #ECEDEF !important;
   --interactive-chip-radius: 32px !important;
   --interactive-chip-shadow: none !important;
   --interactive-answer-choice-border: var(--interactive-accent-color) !important;
-  --interactive-answer-choice-background: white !important;
+	--interactive-answer-choice-background: white !important;
+	--interactive-prompt-text-size: 22px;
```

Comment:
Are we OK with publishers being able to override this?

Let's do:
- if overridable: `--interactive` prefix
- if not overridable: `--i-amphtml-interactive` prefix, publisher can't write `i-amphtml` because it's invalid AMP

Reply:
Oh, this one specifically we should mark as important, but I'll do that as well